src/components/shoppingLists.js

```diff
@@ -25,13 +26,14 @@ export class ShoppingLists extends Component {
             onClick={() => this.clickedAList(list.name)}
             to={`/lists/${list.id}`}
           >
-            Name: {list.name} Address: {list.address}
+            {list.name}
```

Comment:
Instead of a <br/> I would wrap the two of these in divs?

Reply:
good call checking now